ptvsd/wrapper.py

```diff
@@ -1726,6 +1775,16 @@ def start_server(port, addhandlers=True):
     server = _create_server(port)
     client, _ = server.accept()
     pydevd = _start(client, server)
+
+    def _wait_for_more_connections():
+        while True:
+            client, _ = server.accept()
+            _add_pydevd_event_handler(client, pydevd, name='ptvsd.Server')
+
+    connection_thread = threading.Thread(target=_wait_for_more_connections,
+                                         name='ptvsd_client_connection')
```

Comment:
Use `ptvsd.Client.Connection` as name, to be consistent with other `ptvsd` thread names